loopd/utils.go

```diff
@@ -19,10 +19,10 @@ func getClient(config *Config, lnd *lndclient.LndServices) (*loop.Client,
 	}
 
 	clientConfig := &loop.ClientConfig{
-		ServerAddress:   config.SwapServer,
-		ProxyAddress:    config.Proxy,
-		Insecure:        config.Insecure,
-		TLSPathServer:   config.TLSPathSwapSrv,
+		ServerAddress:   config.Server.Host,
```

Comment:
I think we should instead pass in the `*loopServerConfig` struct directly.

Reply:
It creates a circular dependency and pushes down the annotated flags even deeper into the stack. Took a brief look at unifying `/` and `/loopd`, but that outgrows the scope of this pr.